main.go

```diff
@@ -178,7 +178,7 @@ func main() {
 
 			logging.SetupDefaultLogger(os.Stdout, slog.LevelDebug)
 
-			node, _, _, _, err := node.InitializeNode(pkString, chainOpts, useDurableStore, durableStoreFolder, msgPort, os.Stdout, peerSlice)
+			node, _, _, _, err := node.InitializeNode(pkString, chainOpts, useDurableStore, durableStoreFolder, msgPort, peerSlice)
```

Comment:
Just checking, but it seems like this change is unrelated to the race condition. Is that correct?

Reply:
Yes it is unrelated to the race condition. Just something I noticed when scanning through the code. Its related to the PR that was merged this morning